### PR TITLE
Migrate existing features and clean up

### PR DESCRIPTION
**Pull request by Hywan**
The goal is to only migrate the existing features from https://github.com/osaris/jekxyl.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+```
+
+Please, read the website to [get more informations about how to
+install](http://hoa-project.net/Source.html).
+
+## Usage
+
+```sh
+$ jekxyl new ./MyProject
+$ jekxyl compile --source ./MyProject/Source --destination ./MyProject/Dist
```
Comment (vonglasow) [inline README.md:19]: il ne faut pas créer le répertoire `Dist` avant ? Chez moi j'ai une erreur car le répertoire n'existe pas ou alors il faut l'inclure dans le process de compilation

We need to create the folder `Dist` before, I have an error when this folder doesn't exist or they must be created in compile process

Comment (Hywan) [inline README.md]: Correct, it must exist before.

Comment (Hywan) [inline README.md:19]: No, the `Dist` directory is created with the `jekxyl new` command.

Comment (vonglasow) [inline README.md:19]: Ok, but currently it's not the case. When running the command `./jekxyl new`, the folder MyProject is created but inside we only have `Source` folder and `Dist` is away

Comment (Hywan) [inline README.md:19]: The `Dist` directory, because empty, was not present in the Git tree. Now it should work. Can you re-test please?

Comment (vonglasow) [inline README.md:19]: Nop, the folder is still not created during the `./jekxyl new MyProject` process.

Comment (vonglasow) [inline README.md:19]: Yep now it's ok

**Comment by Hywan** [comment]
Closed by https://github.com/jekxyl/jekxyl/commit/6efe3b8018c697b64cb9a2026072db4d41b4fcb4.

**Pull request closed without merging**